src/index.js

```diff
@@ -1059,7 +1059,16 @@ const _start = () => {
     if (u === '.') {
       console.warn('NOTE: You ran `exokit . <url>`\n(Did you mean to run `node . <url>` or `exokit <url>` instead?)')
     }
-    u = u.replace(/^exokit:/, '');
+    if (u.includes("exokit://")){
```

Comment:
Probably should be regexes. It's not unthinkable that a URL would be in a query string, which would throw all of this off.

Reply:
> It's not unthinkable that a URL would be in a query string

What would that look like? `someurl.com?h=exokit://othersite.com`

Reply:
More likely it would be `http://`, but yes.